lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -195,29 +195,39 @@ class OffscreenImages extends ByteEfficiencyAudit {
     }, /** @type {Map<string, WasteResult>} */ (new Map()));
 
     const settings = context.settings;
-    return artifacts.requestInteractive({trace, devtoolsLog, settings}).then(interactive => {
-      const unfilteredResults = Array.from(resultsMap.values());
+
+    let items;
+    const unfilteredResults = Array.from(resultsMap.values());
+    // get the interactive time or fallback to getting the end of trace time
+    try {
+      const interactive = await artifacts.requestInteractive({trace, devtoolsLog, settings});
+
+      // use interactive to generate items
       const lanternInteractive = /** @type {LH.Artifacts.LanternMetric} */ (interactive);
       // Filter out images that were loaded after all CPU activity
-      const items = context.settings.throttlingMethod === 'simulate' ?
+      items = context.settings.throttlingMethod === 'simulate' ?
         OffscreenImages.filterLanternResults(unfilteredResults, lanternInteractive) :
         // @ts-ignore - .timestamp will exist if throttlingMethod isn't lantern
         OffscreenImages.filterObservedResults(unfilteredResults, interactive.timestamp);
+    } catch (err) {
+      // use end of trace as a substitute for finding interactive time
+      items = OffscreenImages.filterObservedResults(unfilteredResults,
```

Comment:
seems like this will try to filter observed results even if it was a lantern case here? is this intentional?

Reply:
Yes, because we wouldn't have a `LH.Artifacts.LanternMetric` object to use since `requestInteractive` is assumed to be the method erroring out, so when we pull the `TraceOfTab` it will be the same for Lantern and for Observed since it is just comparing the wasted artifacts and the timestamp gotten from `TraceOfTab`.

That was my logic at least, dunno if that makes sense?

Reply:
Hm, yeah I'm just not sure this case will ever come up in Lantern world since the ByteEfficiencyAudit base class will be needing all the same things that would have errored `requestInteractive` to compute the savings. Maybe worth adding a comment that this is specifically to address the observed case then and bail on lantern?

I might be behind the times though and perhaps there was a specific lantern-case that inspired #5955 I'm not aware of, happy to learn what was going on there too if that's the case :)

Reply:
@paulirish was this ever including a lantern error?  

I think it was just for the Observed case, but logically it catches both cases and funnels them into the same method since in theory I think both _could_ be mapped to End of Trace?  Not sure if they should, but that's what I imagined & implemented.

Reply:
Got more context from @brendankenny and made it so that when Lantern throws an error getting interactive the error is rethrown!